common/autoware_perception_rviz_plugin/CMakeLists.txt

```diff
@@ -2,6 +2,11 @@ cmake_minimum_required(VERSION 3.14)
 project(autoware_perception_rviz_plugin)
 
 find_package(autoware_cmake REQUIRED)
+find_package(autoware_perception_msgs REQUIRED)
+find_package(pluginlib REQUIRED)
+find_package(rviz_common REQUIRED)
+find_package(rviz_default_plugins REQUIRED)
+find_package(visualization_msgs REQUIRED)
```

Comment:
Something seems off with the necessity of this change. When I compiled this yesterday from source when some of the deps (for ex: `pluginlib`, `rviz_common`, `visualization_msgs`) were not installed, compilation wouldn't succeed first time obviously because the deps weren't available. But the build would fail even after installing those packages.

Only after `find_package`ing them did the build succeed. Now, when I try building this package without any of these changes, compilation succeeds just fine!

Reply:
In principle, since `autoware_perception_msgs`, `rviz_common` and `rviz_default_plugins` are listed as `<depend>` in `package.xml`, they should be `find_package`d?